### DISCUSSION TITLE
How do you get the actual name of a player?

### DISCUSSION THREAD

**Discussion by GEOEGII555** [adepierre/Botcraft#152] [Q&A]
I'm on a server where players have a custom prefix/suffix, and when I use the GetPlayerName function to find out who sent a message it also includes the prefix/suffix. How do I get the actual player name? The prefix can be changed.

**Comment by adepierre** [comment]
Hello, Botcraft only tracks the data sent by the server. So I don't know how does the server send names with prefix/suffix, but if the data don't explicitly say "this is the prefix", "this is the name" and "this is the suffix" then Botcraft can't know about them.

I guess you could try to send a request to Mojang API with the player UUID to get its official username, but you'd have to implement that on your side.
